tools/reproduce_comparison/compareMacOS.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -e
+
+help()
```

Comment:
I think there's an std shell/unix command line pattern to use in help to indicate mandatory or optional parameters, e.g.:

`usage: cp [-R [-H | -L | -P]] [-fi | -n] [-aclpsvXx] source_file target_file`